pages/common/nmap.md

```diff
@@ -26,3 +26,11 @@
 - Perform TCP and UDP scanning (use -sU for UDP only, -sZ for SCTP, -sO for IP):
 
 `nmap -sSU {{address_or_addresses}}`
+
+- Perform scan using an nmap nse script:
```

Comment:
I think this example is a bit corner case. I am okay with adding the next example.

Reply:
After reading the review of my ffmpeg pull I was actually just thinking that the generic "any nmap script" example probably wasn't enirely fitting. Will update this pull accordingly.